web/crux/src/app/registry/registry.mapper.ts

```diff
@@ -93,6 +96,7 @@ export default class RegistryMapper {
         apiUrl: null,
         token: null,
         user: null,
+        namespace: RegistryNamesapaceEnum.unknown,
```

Comment:
why not just null?

Reply:
I think it's okay. We talked with Peti about this, and came to the conclusion unknown is much better.

Reply:
we talked this with @polaroi8d in person, but i can make the field nullable if necessary